hanyu/settings_base.py

```diff
@@ -19,11 +19,6 @@
 # Quick-start development settings - unsuitable for production
 # See https://docs.djangoproject.com/en/1.10/howto/deployment/checklist/
 
-# SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = '91oqzf9f0(*vkge+t65359da_jdl^fdqv)*g8_edjaqdr83nk&'
```

Comment:
Thanks.  I kept the original key in my development environment, but defined an environment variable with a new key for the production environment.